rest_framework/pagination.py

```diff
@@ -264,6 +264,12 @@ def get_page_size(self, request):
 
         return self.page_size
 
+    def get_page(self, request):
```

Comment:
I think I'll change the name a bit:
```suggestion
    def get_page_number(self, request):
```

Reply:
Yep, that's a good idea - will do.